trailmap/acquisition_paths.py

```diff
@@ -0,0 +1,42 @@
+import networkx as nx
+from pprint import pprint
+
+
+def conduct_apa(facility_dict_in, facility_dict_out):
+    ''' '''
+    G = build_graph(facility_dict_in, facility_dict_out)
+
+    sources = list(node for node, in_deg in G.in_degree() if in_deg == 0)
+    sinks = list(node for node, out_deg in G.out_degree() if out_deg == 0)
+
+    print("\nSimple paths")
+    pathways = find_simple_paths(G, sources, sinks)
+    print(pathways)
```

Comment:
are those print statement required ? (or what is just for debugging ?)

Reply:
If the print statements are intended for something other than debugging... then we should discuss separation of calculation from I/O.  Ideally, this method should just return the pathways and let another method take care of output

Reply:
I have separated the two, I think? `find_simple_paths` calculates the pathways and returns them here (line 13). `conduct_apa` (this function) just calls the methods that build the graph and find pathways and then prints the output. Are you recommending I create an additional function beyond `conduct_apa` that is used just for printing?